### PR TITLE
Add A RuleGroup to the Global Policy.  Useful for FMS cases.

### PR DESCRIPTION
**Pull request by pkennedyvt**
*Issue #, if available:*

*Description of changes:*
Added a Rule group for CLOUDFRONT policy, which can be helpful when used in conjunction with FMS.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by aijunpeng** [comment]
Thanks for submitting the PR. We added it to our backlog for evaluation in a future release.

File: deployment/aws-waf-security-automations-webacl.template [hunk 1]
```diff
@@ -604,6 +605,113 @@ Resources:
...
+      VisibilityConfig: 
+        SampledRequestsEnabled: true
+        CloudWatchMetricsEnabled: true
+        MetricName: !Join ['', [!Join ['', !Split ['-', !Ref ParentStackName]], 'WAFSecAutomationsSecurityGroup']]
+      Capacity: 10
+      Tags:
+        - Key: Division
+          Value: "ITRMS"
+        - Key: Consumer
+          Value: "gsw-cdn-eng@merck.com"
```
Comment (diimpp) [inline deployment/aws-waf-security-automations-webacl.template:623]: Is it your company data? Should it really be included in PR?

**Comment by pkennedyvt** [comment]
I thought I had santitized everything, that should not be.   It’s a dummy account, but still should come out.

--pk

**Comment by aijunpeng** [comment]
This is not added to roadmap after evaluation. Close the old ticket.

**Pull request closed without merging**